stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -102,12 +239,31 @@ extension DistributedActor {
 // ==== Codable conformance ----------------------------------------------------
 
 extension CodingUserInfoKey {
+
+  /// Key which is required to be set on a `Decoder`'s `userInfo` while attempting
+  /// to `init(from:)` a `DistributedActor`. The stored value under this key must
+  /// conform to ``DistributedActorSystem``.
+  ///
+  /// Missing to set this key will result in that initializer throwing, because
+  /// an actor system is required
```

Comment:
Probably "Failing to set" and missing the rest of the sentence?

Reply:
thanks